custom/icds_reports/templates/icds_reports/icds_app/mobile/program-summary.directive.html

```diff
@@ -1,16 +1,38 @@
-{% load i18n %}
-<div cg-busy="[$ctrl.myPromise, $rootScope.locationPromise]">
-    <div class="row no-margin">
-      <h1 ng-bind="$ctrl.label"></h1>
-    </div>
-    <div class="row">
-        <div class="dot-menu">
-            <div class="dot-menu-item pointer" ng-repeat="step in $ctrl.steps">
-                <dot-link label="{$ step.label $}" route="{$ step.route $}"></dot-link>
+{% load staticfiles %}
+<div class="app-bg h-100vh overflow-hidden">
```

Comment:
most of this copied from https://github.com/dimagi/mobile-dashboard-prototype/pull/8